src/Dropdown/Dropdown.tsx

```diff
@@ -65,10 +90,23 @@ const Dropdown: React.FC<DropdownProps> = ({
                 onClick={onClick}
                 disabled={disabled}
             >
-                <span>{selected?.label}</span>
+                <span>
+                    {items.findIndex(e => e.value === selected.value) === -1
+                        ? ''
+                        : selected.label}
```

Comment:
Unless I misunderstand, this is simply to ensure the value is a valid one?
I don't see a scenario where that could be the case though.
Could you explain?

This can be shortened a bit either way (just personal preference)
`items.find(e => e.value === selected.value) ? '' : selected.label`

Reply:
Given that the data is set by the parent component, there might be the case that the user enter a new DropownItem object such as {value, key} and:

- Since it is a different obj from the on in the items we can only match the value to ensure that it is the same one
- Since it is a different obj it can not even be in the list of items

Hence I felt the need to do data validation.